project/SotaBuild.scala

```diff
@@ -76,6 +77,25 @@ object SotaBuild extends Build {
     }
   )
 
+  lazy val nexusPassword =
+    sys.env
+      .get("NEXUS_PASSWORD")
+      .orElse(sys.props.get("nexus.password"))
+      .getOrElse("")
+
+  lazy val nexusPublish = Seq(
+    credentials += Credentials("Sonatype Nexus Repository Manager",
+      "nexus.prod01.internal.advancedtelematic.com",
```

Comment:
The changes in `rvi_sota_server` will be pushed upstream, so repository uri should be read from env, as username and password